.github/workflows/benchmarks_gitlab.yml

```diff
@@ -22,20 +23,27 @@ jobs:
           toolchain: nightly
           override: true
 
-      - name: Set bench file
+      - name: Rust Cache
+        uses: Swatinem/rust-cache@v2.0.0
+
+      - name: Copy bench results
         id: step_one
         run: |
-          echo "bench_file=bench/$(date "+%d-%m-%Y")/output.txt" >> $GITHUB_ENV
+          cp bench/$(date "+%d-%m-%Y")/output.txt output.txt
+
+      - name: Switch branch
+        id: step_two
+        run: |
+          git checkout master
```

Comment:
apparently the github action does some git magic under the hood and doesn't work if we are already on gh-pages branch